src/common/internal-plugins.js

```diff
@@ -0,0 +1,110 @@
+"use strict";
+
+const __r = require;
```

Comment:
A comment here indicating the babel plugin that looks for this identifier would be good to have.

Reply:
Yeah I just wasn't sure if we should do this or keep `eval("require")`.

Reply:
We could name it `externalRequiredThatGetsTransformedByBabel` 😉